config/locales/errors/en.yml

```diff
@@ -90,6 +90,9 @@ en:
         phone call option below, or use your personal key.
       pwned_password: The password you entered is not safe. It’s in a list of known
         passwords exposed in data breaches.
+      too_short:
```

Comment:
Unfortunately this is the catch-all translation key for ActiveModel length, so it will also apply to non-password attributes. Example of it saying password on a non-password field:

```ruby
form = OpenidConnectLogoutForm.new({})
[13] pry(main)> form.valid?
=> false
[14] pry(main)> form.errors.messages
=> {:redirect_uri=>["redirect_uri does not match registered redirect_uri"],
 :id_token_hint=>["Please fill in this field.", "id_token_hint was not recognized"],
 :post_logout_redirect_uri=>["Please fill in this field."],
 :state=>
  ["Please fill in this field.",
   "This password is too short (minimum is 22 characters)"]}
```